internal/testing/setup/setup.go

```diff
@@ -334,3 +334,11 @@ func NewGCPDialOptions(t *testing.T, recording bool, filename string) (opts []gr
 	}
 	return opts, done
 }
+
+// RunTestsDependingOnDocker returns true when either:
+// 1) Not on Travis.
+// 2) On Travis Linux environment, where Docker is available.
+func RunTestsDependingOnDocker() bool {
```

Comment:
I feel the name of this function could be better, conveying it's a predicate. Something like `HasDockerTestEnvironment` or `CanRunLocalServerTests`, etc.

Reply:
Went with `HasDockerTestEnvironment`